src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1522,6 +1522,22 @@ static bool processSubscriptions
                                                  tenant,
                                                  xauthToken))
     {
+      long long rightNow = getCurrentTime();
+
+      //
+      // If broker running without subscription cache, put lastNotificationTime and count in DB 
+      //
+      if (mongoSubCacheActive == false)
+      {
+        BSONObj query  = BSON("_id" << OID(mapSubId));
+        BSONObj update = BSON("$set" <<
+                              BSON(CSUB_LASTNOTIFICATION << rightNow) <<
+                              "$inc" << BSON(CSUB_COUNT << 1));
+        
+        collectionUpdate(getSubscribeContextCollectionName(tenant), query, update, false, err);
```

Comment:
Ah, make the function return false ...
Sure, that's not a problem. Not sure that anybody looks at the return value though ... :-)